st2common/st2common/models/api/rule.py

```diff
@@ -53,6 +69,49 @@ class ActionSpec(BaseAPI):
     v.update({'required': True})
 
 
+class RuleTypeAPI(BaseAPI):
+    model = RuleTypeDB
+    schema = {
+        'title': 'RuleType',
+        'description': 'A specific type of rule.',
+        'type': 'object',
+        'properties': {
+            'id': {
+                'description': 'The unique identifier for the action runner.',
+                'type': 'string',
+                'default': None
+            },
+            'name': {
```

Comment:
Looks like you follow similar model as `PolicyType` follows.

That's good, Ideally we would eventually try to refactor some of that functionality and conventions in a base class so everyone can follow the same established pattern.